internal/model/json/validation/typedefs/fields.go

```diff
@@ -217,4 +217,44 @@ var Reference = &generator.Schema{
 	Pattern:     ReferencePattern,
 }
 
+var PemKey = &generator.Schema{
+	Type: "object",
+	Properties: map[string]*generator.Schema{
+		"private_key": {Type: "string"},
```

Comment:
You can use [some existing formats ](https://github.com/Kong/koko/blob/main/internal/model/json/schema/formats.go#L71-L73)for these

Example: https://github.com/Kong/koko/blob/main/internal/resource/certificate.go#L180